coreless-ui/frontend/src/views/forms/party-routing-profile/PartyRoutingProfile.js

```diff
@@ -18,31 +18,44 @@ export default class PartyRoutingProfile extends React.Component {
 
     this.state = {
       dataArray: [],
+      dataMap: new Map(),
       error: '',
     }
   }
   async componentDidMount() {
-    await fetch('/partyRoutingProfile')
+    await fetch('/getPartyRoutingProfileKeys')
```

Comment:
can't you make it work asynchronously? I don't like the await that blocks the thread.

Reply:
fixed, thank you